src/pencilkit.cs

```diff
@@ -49,8 +55,19 @@ enum PKInkType {
 		Marker,
 	}
 
+	[iOS (14, 0), NoMac]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
+	[Native]
+	enum PKCanvasViewDrawingPolicy : ulong
+	{
+		Default,
+		AnyInput,
+		PencilOnly,
+	}
+
 	[Unavailable (PlatformName.UIKitForMac), Advice ("This API is not available when using UIKit on macOS.")]
 	[iOS (13, 0), NoMac]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
```

Comment:
you need to remove the `[Unavailable (PlatformName.UIKitForMac), Advice ("This API is not available when using UIKit on macOS.")]` above since it contradict your change

`PlatformName.UIKitForMac` == `PlatformName.MacCatalyst`, just a new name :)